HKMP/Api/Client/IClientManager.cs

```diff
@@ -90,4 +90,14 @@ public interface IClientManager {
     /// Event that is called when another player leaves the local scene.
     /// </summary>
     event Action<IClientPlayer> PlayerLeaveSceneEvent;
+
+    /// <summary>
+    /// Event that is called when HKMP modifies the game's time scale.
+    /// </summary>
+    delegate void SetTimeScale(float timeScale);
```

Comment:
Is it necessary to make a delegate here? Since this is a simple delegate with a `float` parameter, can it not simply be as `Action<float>` in the event below?